app/assets/stylesheets/pages/_index.scss

```diff
@@ -1,3 +1,2 @@
 // Import page-specific CSS files here.
 @import "home";
-@import "studios/index";
```

Comment:
you removed the style here. Make sure to merge main into your branch to get the latest updates